hs-abci-examples/nameservice/docker-compose-test.yaml

```diff
@@ -17,40 +17,28 @@ services:
     ports:
       - "26656-26657:26656-26657"
   nameservice:
-    entrypoint: /usr/local/bin/nameservice
     build:
       context: ../../.
       dockerfile: Dockerfile
     image: hs-abci:test
     environment:
-     - ES_HOST=elk
-     - ES_PORT=9200
-     - STATS_PORT=5555
+     - STATS_PORT=9200
     restart: always
-    depends_on:
-      - elk
-      - datadog
+    entrypoint: /usr/local/bin/nameservice
     expose:
       - "26658"
-      - "5555"
+      - "9200"
   datadog:
     image: datadog/agent:latest
+    environment:
+     - DD_API_KEY=${DD_API_KEY}
     volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - /proc/:/host/proc/:ro
      - /sys/fs/cgroup:/host/sys/fs/cgroup:ro
     labels:
       com.datadoghq.ad.check_names: '["openmetrics"]'
       com.datadoghq.ad.init_configs: '["{}"]'
-      com.datadoghq.ad.instances: '["{\"prometheus_url\":\"http://%%host%%:5555/metrics \",\"namespace\":\"nameservice\",\"metrics\":[\"count_buy\" \"count_set\" \"count_delete\" \"histogram_buy*\" \"histogram_set*\" \"histogram_delete*\"]}"]'
-  elk:
-    image: sebp/elk:683
-    ports:
-      - "5601:5601"
-      - "9200:9200"
-      - "5044:5044"
-    expose:
-      - "9200"
-      - "5601"
+      com.datadoghq.ad.instances: '["{\"prometheus_url\":\"http://%%host%%:9200/metrics \",\"namespace\":\"nameservice\",\"metrics\":[\"count_buy\" \"count_set\" \"count_delete\" \"histogram_buy*\" \"histogram_set*\" \"histogram_delete*\"]}"]'
```

Comment:
How come you're cherry-picking metrics here, but in the other docker-compose with the YAML config you just `metrics: ["*"]`?

Reply:
this actually shouldn't even be here, I wanted to remove all logging and metrics from the test compose file